Navigation

parvez-rashid opened the pull request:
View the screenshot below.

![image](https://user-images.githubusercontent.com/84724590/153996813-cb82c7b5-bbab-46f5-98ea-e2e3eb12b55c.png)

parvez-rashid committed:
Feat: changed nav display to flex

parvez-rashid committed:
Feat: Added icons and expanded drop down

Added icons to nav items and listed all components in the drop down.

parvez-rashid committed:
Style: Added bg color to navbar

Added background color to navbar and used variables for colors.

Naman-Saxena1 commented:
Preview link is not working, code looks fine

parvez-rashid commented:
> Preview link is not working, code looks fine

Will include screenshots for ease. I'l fix the preview link soon.

Naman-Saxena1 commented on navigation.html:
```diff
@@ -0,0 +1,48 @@
+        </li>
+        <li>
+          <i class="fa-brands fa-github"></i>
+          <a class="options link" href="/navigation.html">github</a>
```
You should add target="_blank", so that github link opens in new tab.

parvez-rashid replied:
Have updated the code. Please approve PR.

parvez-rashid committed:
Feat: Added target blank to link

Added `target: "_blank"` attribute to GitHub link so that it opens in a new tab.

Ankur9669 commented on navigation.html:
```diff
@@ -0,0 +1,48 @@
+      src="https://kit.fontawesome.com/43943f2889.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>Document</title>
```
Use script tag at the bottom

parvez-rashid replied:
Have done it, thanks for the review.

Ankur9669 reviewed (approved):
Code looks great to me!!!!. Nice work

parvez-rashid committed:
Refactor: moved script tag to bottom

Naman-Saxena1 reviewed (approved):
Looks fine now 👍
